### PR TITLE
Milestone 3 - Contact Form and Footer

### PR DESCRIPTION
**Pull request by t-bello7**
- Add and style the contact from 
- Add and style the footer containing social media icons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,123 +1,191 @@
...
+        </ul>
+      </div>
+    </section>
+    <footer class="container"> 
+      <div class="contact-form mg-8 ">
+        <h2 class="center ff-inter">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form class="mg-2 flex contact" action="https://formspree.io/f/mwkywooe">
```
Comment (danmainah) [inline index.html]: - [x] Kindly add the `post` attribute to your form, currently, data is not sent to the form spree server.

File: index.html [hunk 2]
```diff
@@ -1,123 +1,191 @@
...
+          <input class="mg-1 contact-form__input ff-inter" type="email" placeholder="Email address" required/>
+          <textarea
+            class="mg-1 contact-form__textarea bg-neutral-1"
+            name="description"
+            id="description"
+            cols="27"
+            rows="10"
+            maxlength="500"
+            placeholder="Enter text here"
+          ></textarea>
```
Comment (danmainah) [inline index.html]: - [x] Kindly add the required attribute to your `<textarea>` tag , as requested in the project requirements.

File: index.html [hunk 3]
```diff
@@ -1,123 +1,191 @@
...
+          <button class="center btn" type="submit">Get in touch</button>
+        </form>
```
Comment (danmainah) [inline index.html]: - [x] Kindly make sure to use flexbox while styling your form, this is one of the project requirements.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @t-bello7 , @chahine-mahdhi  and @aliatayee , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 3 above)
Comment (t-bello7) [reply index.html]: Yes I already used flexbox on the form styling
Screenshot from 2022-04-27 07-35-40
.

Comment (t-bello7) [reply index.html]: Thank you for the review.

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- Kindly ensure that as  partners you always have balanced commits in pair programming project 👍 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**